lecture_1/homework/app/controllers/api/v1/clans_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Api
+  module V1
+    class ClansController < ApplicationController
+      before_action :default_limit, only: %i[index]
```

Comment:
Wydaje mi się, że bardziej elegancko by to wyglądało, gdybyś zamiast callbacka ustawiającego wartość dla `params[:limit]`, przetwarzał tą wartość w metodzie `limit` np. tak

```ruby
def limit
  @limit ||= params[:limit] || 20
end
```

Reply:
Poprawione :D